workshops/forms.py

```diff
@@ -70,26 +125,11 @@ def __init__(self, form=None):
         self.inputs.append(Submit('', 'Submit'))
 
 
-class BootstrapHelperWiderLabels(BootstrapHelper):
-    """SWCEventRequestForm and DCEventRequestForm have long labels, so this
-    helper is used to address that issue."""
-    label_class = 'col-lg-3'
-    field_class = 'col-lg-7'
-
-
 class BootstrapHelperFormsetInline(BootstrapHelper):
     """For use in inline formsets."""
     template = 'bootstrap/table_inline_formset.html'
 
 
-bootstrap_helper = BootstrapHelper()
-bootstrap_helper_get = BootstrapHelperGet()
-bootstrap_helper_with_add = BootstrapHelperWithAdd()
-bootstrap_helper_filter = BootstrapHelperFilter()
-bootstrap_helper_wider_labels = BootstrapHelperWiderLabels()
-bootstrap_helper_inline_formsets = BootstrapHelperFormsetInline()
```

Comment:
Well yes, but it's easier to have one class/instance `FormHelperFilter` that you know you can use when you're writing a new filter form class -- you're then safer, because you're using a predefined "styles" (or arguments to the `BootstrapHelper`), the very same that you're using in every other filter form.